### PR TITLE
fix: fix some small bug

### PR DESCRIPTION
**Pull request by likyh**
### Summary
1. hide time filter when bp advanced mode
2. set settings=null when advance mode
3. run linter
4. show status as default bp detail page.

### CODE REVIEW

File: config-ui/src/pages/blueprint/create/step-four/index.tsx [hunk 1]
```diff
@@ -22,9 +22,11 @@ import { SyncPolicy } from '@/plugins'
 import { Card, Divider } from '@/components'
 
 import { useCreateBP } from '../bp-context'
+import {ModeEnum} from "@/pages/blueprint/create/types";
```
Comment (mintsweet) [inline config-ui/src/pages/blueprint/create/step-four/index.tsx:25]: `../types`

File: config-ui/src/plugins/common/sync-policy/index.tsx [hunk 2]
```diff
@@ -17,26 +17,21 @@
...
-  Checkbox,
-  Position
-} from '@blueprintjs/core'
+import { Checkbox, Icon, InputGroup, Position, Radio, RadioGroup } from '@blueprintjs/core'
 import { Popover2 } from '@blueprintjs/popover2'
 
 import StartFromSelector from '@/components/blueprints/StartFromSelector'
 import { getCron, getCronOptions } from '@/config'
 import CronHelp from '@/images/cron-help.png'
+import { ModeEnum } from "@/pages/blueprint/create/types";
```
Comment (mintsweet) [inline config-ui/src/plugins/common/sync-policy/index.tsx:26]: Can be deleted if not used.

**Review by mintsweet — DISMISSED** [review]

**Review by klesh — APPROVED** [review]
reviewed by @mintsweet

**Review by klesh — APPROVED** [review]
reviewed by @mintsweet

**Pull request merged**